infrastructure/ci/dev-platform/shared/create-sse-frontend-pipeline.sh

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env bash
```

Comment:
This can be replaced with a more general script.

Reply:
removed pipeline script to use dev platform provided script